src/events/interactionCreate/roles.js

```diff
@@ -1,39 +1,42 @@
-//assigns a role when a button is pressed for the role
+// assigns a role when a button is pressed for the role
```

Comment:
Can this file be deleted, or it being used as an example?

Reply:
It can be deleted, I missed it.